db/migrations/0002_alter_player_guild_alter_player_race_and_more.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 4.0.2 on 2024-11-26 18:44
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='player',
+            name='guild',
+            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='players', to='db.guild'),
+        ),
+        migrations.AlterField(
+            model_name='player',
+            name='race',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='players', to='db.race'),

```

Comment:
Ensure that the `to` parameter 'db.race' matches the actual app label and model name.